source/api/extension/navplace/index.md

```diff
@@ -0,0 +1,331 @@
+      "features":[
+         {
+            "id": "http://example.com/feature/1",
+            "type": "Feature",
+            "properties":{},
+            "geometry":{
+               "type": "Point",
+               "coordinates":[
+                  9.938,
+                  51.533
+               ]
+            }
+         }
+      ]
+   }
+}
+```
+
+
+The `navPlace` property can be used with the [IIIF Presentation 3 API Defined Types](https://iiif.io/api/presentation/3.0/#21-defined-types).
+
+
+
+*   A Collection _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Collection.
+*   A Manifest _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Manifest.
+*   A Range _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Range.
+*   A Canvas _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Canvas.
+*   Other types of resource _MUST NOT_ have the `navPlace` property.<br/>
+   Clients _SHOULD_ ignore `navPlace` on other types of resource.
+
+The `navPlace` property's value follows a specific pattern.
+
+
+
+*   The value _SHOULD_ be an embedded Feature Collection. However, the value _MAY_ be a referenced GeoJSON Feature Collection. 
+*   Feature Collections referenced in the `navPlace` property _MUST_ have the `id` and `type` properties. The reference object _MUST NOT_ have the `features` property, such that clients are able to recognize that it should be retrieved in order to be processed. <br/>
+```json-doc
+{"navPlace":{"id": "https://example.org/iiif/1/feature-collection", "type": "FeatureCollection"}}
+```
+
+
+### 2.2 GeoJSON
+
+This extension utilizes the [GeoJSON specification](https://tools.ietf.org/html/rfc7946). Values for the `navPlace` property follow the conventions for [GeoJSON Feature Collections](https://datatracker.ietf.org/doc/html/rfc7946#section-3.3).
+
+
+#### 2.2.1 GeoJSON as Linked Data
+
+[GeoJSON-LD](https://geojson.org/geojson-ld/) is a publicly available vocabulary and linked data context for the GeoJSON specification. The `navPlace` extension context file contains this context and reference to the vocabulary which ensures the geographic data values are well described. The example below shows how to set the `@context` property for IIIF resources with the `navPlace` property. [Section 3](#3-linked-data) has more detail on linked data compatibility.
+
+
+
+```json-doc
+{
+    "@context":[ 
+       "http://iiif.io/api/extension/navplace/context.json",
+       "http://iiif.io/api/presentation/3/context.json"
+    ]
+}
+```
+
+
+#### 2.2.2 Feature
+
+A Feature represents a spatial bounding. Every Feature is a GeoJSON object. [Go to the GeoJSON specification for a visual example](https://datatracker.ietf.org/doc/html/rfc7946#section-1.5). In this document, the term "Feature" relates to "Feature Object" from the GeoJSON specification.
+
+
+*   A Feature _MUST NOT_ be NULL.
```

Comment:
I don't think this is necessary. It's required to be a JSON object, so it clearly cannot be a null.